util/blockbreaker.py

```diff
@@ -24,7 +24,7 @@ def process_options():
                         dest = "config",
                         required = True,
                         help = "Configuration type to get from the json file",
-                        choices = [ "benchmarks", "mv-params", "tool-params", "passthru-args", "tags", "endpoint", "endpoints" ])
+                        choices = [ "benchmarks", "mv-params", "tool-params", "run-parameters", "tags", "endpoint", "endpoints" ])
```

Comment:
nit... in case you need a respin, I'd prefer "runtime-params" to be more like the others

Reply:
Hmm, I could see using `run-params` as that would align with the other values and now that you mention it that kinda seems like a good idea.  IMHO, 'runtime' kinda implies something about the actual benchmark run execution time which is not really what these parameters are about.